functions/Parse-DbaCsvMetadata.ps1

```diff
@@ -0,0 +1,360 @@
+﻿function Get-UserPrompt {
+    param (

```

Comment:
Please move this function to your BEGIN block. All  files should start with your initial function name and be self contained (except for shared functions); please take a look at other commands such as [Copy-SqlLogin.ps1](https://github.com/sqlcollaborative/dbatools/blob/master/functions/Copy-SqlLogin.ps1) as an example. Placing functions in the BEGIN block makes them private to your script.